PlatformUI/EventListener/PlatformUIListener.php

```diff
@@ -19,10 +19,6 @@ protected function isPlatformUIRequest(Request $request)
             return false;
         }
 
-        if (!$request->headers->has('X-PJAX') || !$request->isXmlHttpRequest()) {
-            return false;
-        }
-
-        return true;
+        return $request->headers->has('X-AJAX-Update') || $request->isXmlHttpRequest();
```

Comment:
well, same answer as in the other comment. Actually, I should have titled this PR *POC* :)